app/src/main/java/com/sun_asterisk/moviedb_49/data/model/Movie.kt

```diff
@@ -0,0 +1,47 @@
+package com.sun_asterisk.moviedb_49.data.model
+
+data class Movie(val build: MovieBuild) {
```

Comment:
https://medium.com/@vicidroiddev/using-builders-in-kotlin-data-class-e8a08797ed56
tham khảo pattern